libs/gi/page-team/src/TeamSetting/TeamExportModal.tsx

```diff
@@ -0,0 +1,333 @@
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+import { useState } from 'react'
+
+// TODO: Translation
+export default function TeamExportModal({
+  show,
+  onHide,
+  teamId,
+}: {
+  show: boolean
+  onHide: () => void
+  teamId: string
+}) {
+  const database = useDatabase()
+  const team = useTeam(teamId)!
+  const { loadoutData } = team
+
+  const [loadoutDataExportSetting, setLoadoutDataExportSetting] =
+    useState<LoadoutDataExportSetting>(() =>
+      loadoutData.map(() => defLoadoutExportSetting())
+    )
+  const onExport = () => {
+    const data = database.teams.export(teamId, loadoutDataExportSetting)
+    const dataStr = JSON.stringify(data)
+    navigator.clipboard
+      .writeText(dataStr)
+      .then(() => alert('Copied team data to clipboard.'))
+      .catch(console.error)
+  }
+  const [selEverything, setSelEverything] = useState(true)
+  const unselectEverything = () => {
+    setSelEverything(true)
+    setLoadoutDataExportSetting(
+      loadoutData.map(() => defLoadoutExportSetting())
+    )
+  }
+
+  const selectEverything = () => {
+    setSelEverything(false)
+    setLoadoutDataExportSetting(
+      loadoutData.map((loadoutDatum) => {
+        if (!loadoutDatum) return defLoadoutExportSetting()
+
+        const loadout = database.teamChars.get(loadoutDatum.teamCharId)
+        if (!loadout) return defLoadoutExportSetting()
+        const { buildIds, buildTcIds, customMultiTargets } = loadout
+        return {
+          convertEquipped: true,
+          convertbuilds: buildIds,
+          convertTcBuilds: buildTcIds,
+          exportCustomMultiTarget: customMultiTargets?.map((_, i) => i) ?? [],
+        }
+      })
+    )
+  }
+
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('md'))
+  return (
+    <ModalWrapper open={show} onClose={onHide}>
+      <CardThemed>
+        <CardHeader
+          title="Team Export"
+          action={
+            <IconButton onClick={onHide}>
+              <CloseIcon />
+            </IconButton>
+          }
+        />
+        <Divider />
+        <CardContent sx={{ display: 'flex', flexDirection: 'column', gap: 1 }}>
+          <Box
+            sx={{
+              display: 'flex',
+              gap: 1,
+              flexDirection: isMobile ? 'column' : 'row',
+            }}
+          >
+            <Alert severity="info">
+              Export the team data to be imported by another user. All the team
+              and loadout data (bonus stats, enemy config, optimize config) are
+              exported. All exported non-TC builds are converted to TC builds.
+            </Alert>
+            <Button
+              color="info"
+              onClick={selEverything ? selectEverything : unselectEverything}
+              startIcon={
+                selEverything ? <CheckBoxIcon /> : <CheckBoxOutlineBlankIcon />
+              }
+            >
+              {selEverything ? 'Select Everything' : 'Unselect Everything'}
```

Comment:
```suggestion
              'Select All'
```
Button text shouldn't change, since the icon is changing